app/javascript/styles/mastodon/components.scss

```diff
@@ -2305,12 +2306,10 @@ button.icon-button.active i.fa-retweet {
 }
 
 .column-header__buttons {
-  position: absolute;
-  right: 0;
-  top: 0;
   height: 100%;
   display: flex;
-  height: 48px;
```

Comment:
`height: 48px;` is needed for mobile view, displayed only back button.
![ws000124](https://user-images.githubusercontent.com/27640522/31851503-0be28392-b6a2-11e7-8bb5-795dd82d6136.png)

Reply:
sorry! fixed in https://github.com/tootsuite/mastodon/pull/5493/commits/ab72670f76c2499eef96338ddba11fa0153d09e7 .